lib/widgets/rules.dart

```diff
@@ -0,0 +1,56 @@
+import 'package:flutter/material.dart';
+
+import '../model/rules.dart';
+
+/// This widget is a text container.
+///
+/// It shows game rules.
+class GameRulesView extends StatelessWidget {
+  const GameRulesView({super.key});
+
+  final double lineSpacing = 15;
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      constraints: const BoxConstraints(maxWidth: 400),
+      child: Column(
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: [
+          Text(
+            'For each board you can choose one of the rules variation to play:',
+            style: Theme.of(context).textTheme.titleLarge,
+          ),
+          const SizedBox(height: 20),
+          Container(
+              decoration: const BoxDecoration(
+                  border: Border(bottom: BorderSide(width: 1))),
+              child: Text(
+                'Traditional rules',
```

Comment:
What's the point to show traditional rules differently? Why not just itemize them as one of the options?

Reply:
Moved to the options list:

![image](https://user-images.githubusercontent.com/64236396/236591863-e42c67c5-ed5f-4ee8-81a5-8b248283624e.png)